### GOAL DEFINITION

# if trajectory is constant or not
bool const_trajectory

# Path Planning OR Trajectory
geometry_msgs/PoseArray path

# Average velocity of the route ([meter]/[second])
float32 average_velocity

# Sampling time ([second])
float32 sampling_time

# if path is already a Trajectory set FALSE
bool make_trajectory

---
### RESULT DEFINITION
int32 distance_traveled_percentage
string mission_status

# Goal distance ([meter])
float32 goal_distance
---
### FEEDBACK DEFINITION
int32 distance_traveled_percentage
string mission_status